src/ui/screens/WebcamCaptureTool.tsx

```diff
@@ -46,15 +53,34 @@ export default class WebcamCaptureTool extends React.Component<any,any> {
     })
   }
 
+  determinePhotoType(dataUri: string): PhotoAttach {
+    const parts: string[] = dataUri.split(",", 2);
+    const type: string = parts[0].split(":")[1].split(";")[0];
+    return {
+      data: parts[1],
+      type,
+      encoding: 'base64'
+    };
+  }
+
   handleUploadPhoto = (dataUri: string) => {
-    const photo = dataUri.replace("data:image/png;base64,","")
+    // This relies on the ImageUpload component returning base64 encoded images
+    // If that ever changes this will superbreak
+    const photo: PhotoAttach = this.determinePhotoType(dataUri);
     this.setState({
       "photo~attach": photo
```

Comment:
Not a critical thing, but using the exact field name here might create problems if we ever have to take a picture that will be stored in a different credential field.